tests/fast-integration/skr-test/test.js

```diff
@@ -33,6 +33,8 @@ describe('SKR test', function() {
   // Run the OIDC tests
   oidcE2ETest(options, getShootInfoFunc);
 
+  oidcE2ETest(options, getShootInfoFunc);
```

Comment:
Is `oidcE2ETest(options, getShootInfoFunc);` intentionally run twice?

Reply:
Yes, but just for the test. At that moment, `skr-upgrade` test was failing, so it was done to check if OIDC is working properly. Eventually, it should be run once.